interface/src/app/mlmodel/mlModelComponents.js

```diff
@@ -30,4 +31,45 @@ angular.module('adage.mlmodel.components', [
   }]
 })
 
+.component('mlModelView', {
+  templateUrl: 'mlmodel/view.tpl.html',
+  controller: ['GlobalModelInfo', function(GlobalModelInfo) {
+    this.modelInfo = GlobalModelInfo;
+  }]
+})
+
+.component('mlModelValidator', {
```

Comment:
Yes, I read these comments, I was just wondering whether this was the best approach, or we should split the functionality of the mlModelValidator into an mlModelValidator service and a component that displays an error message. I am ok with this approach if you feel it is the best way.